flight/Modules/UAVOMavlinkBridge/UAVOMavlinkBridge.c

```diff
@@ -125,7 +123,8 @@ static int32_t uavoMavlinkBridgeInitialize(void) {
 		module_enabled = true;
 		updateSettings();
 
-		serial_buf = PIOS_malloc(MAVLINK_MAX_PACKET_LEN);
+		mav_msg = PIOS_malloc(sizeof(*mav_msg));
```

Comment:
Might be a good idea to ensure that the malloc succeeded (and same for the existing one right below) and disable the module if not?